core/src/main/scala/kafka/raft/RaftManager.scala

```diff
@@ -220,25 +220,14 @@ class KafkaRaftManager[T](
   }
 
   private def buildMetadataLog(): KafkaMetadataLog = {
-    val defaultProps = LogConfig.extractLogConfigMap(config)
```

Comment:
I see we are no longer calling this, but rather explicitly populating the log properties. Are there other log configs we might want to expose?

Reply:
It seemed best for now to not allow overrides since this is a new usage of `Log` and we haven't had time to understand the impact of all of the configurations for this usage. Most of them are probably safe, but others do not even make sense (e.g. the retention settings). Let me open a JIRA so that we can consider which configs we want to expose for the metadata log and how we want to expose them.

Reply:
I filed this JIRA: https://issues.apache.org/jira/browse/KAFKA-12423.